src/Standard.Licensing.Tests/LicenseValidationTests.cs

```diff
@@ -56,23 +52,23 @@ public void Can_Validate_Valid_Signature()
                                   <Signature>MEUCIQCCEDAldOZHHIKvYZRDdzUP4V51y23d6deeK5jIFy27GQIgDz2CndjBh4Vb8tiC3FGQ6fn3GKt8d/P5+luJH0cWv+I=</Signature>
                                 </License>";
 
-            var license = License.Load(licenseData);
+            License? license = License.Load(licenseData);
 
-            var validationResults = license
-                .Validate()
-                .Signature(publicKey)
-                .AssertValidLicense();
+            IEnumerable<IValidationFailure>? validationResults = license
+                                                                 .Validate()
+                                                                 .Signature(publicKey)
+                                                                 .AssertValidLicense();
 
-            Assert.That(validationResults, Is.Not.Null);
-            Assert.That(validationResults.Count(), Is.EqualTo(0));
+            IEnumerable<IValidationFailure> validationFailures = validationResults.ToList();
+            Assert.That(validationFailures, Is.Not.Null);
+            Assert.That(validationFailures.Count(), Is.EqualTo(0));
```

Comment:
What is the purpose of creating the list here, the old code contained a single enumeration of `validationResults`, now with this code there is the enumeration to produce the list and then a second enumeration in the assertion.